rest/route/service.go

```diff
@@ -188,6 +188,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/tasks/{task_id}").Version(2).Patch().Wrap(requireUser, addProject, editTasks).RouteHandler(makeModifyTaskRoute())
 	app.AddRoute("/tasks/{task_id}/annotations").Version(2).Get().Wrap(requireUser, viewAnnotations).RouteHandler(makeFetchAnnotationsByTask())
 	app.AddRoute("/tasks/{task_id}/annotation").Version(2).Put().Wrap(requireUser, editAnnotations).RouteHandler(makePutAnnotationsByTask())
+	app.AddRoute("/tasks/annotations").Version(2).Patch().Wrap(requireUser, editAnnotations).RouteHandler(makeBulkCreateAnnotations())
```

Comment:
If this is a patch route, then the word "create" here doesn't feel fully accurate, since it can potentially be an update.